manager/controllers/app/blueprint_controller_test.go

```diff
@@ -24,7 +24,8 @@ func deployBlueprint(namespace string, shouldSucceed bool) {
 
 	// Set the correct namespace
 	blueprint.SetNamespace(namespace)
-	fmt.Printf("Blueprint controller unit test - blueprint namespace: %s\n", namespace)
+	blueprint.Spec.ModulesNamespace = namespace
```

Comment:
Use  `utils.GetDefaultModulesNamespace() `here. The `namespace` variable indicates where the blueprint is deployed (should be system namespace for positive behavior, something else for negative behavior). It is not modules' namespace.

Reply:
Yup good catch; changed